modules/core/src/lib/layer.js

```diff
@@ -65,9 +65,9 @@ const defaultProps = {
   onDataLoad: {type: 'function', value: null, compare: false, optional: true},
   fetch: {
     type: 'function',
-    value: (url, {propName, layer}) => {
+    value: (url, {propName, layer, signal}) => {
       const {resourceManager} = layer.context;
-      const loadOptions = layer.getLoadOptions();
+      const loadOptions = signal ? {...layer.getLoadOptions(), signal} : layer.getLoadOptions();
```

Comment:
The `signal` parameter is only used by the `TileLayer`, which should not use `resorceManager`.

If we start using AbortSignal as a generic solution in the future, this line still breaks the `resourceManager` logic below.

I recommend overriding the `fetch` implementation in `TileLayer.defaultProps` to avoid unexpected breakage.

Reply:
Can we do this piece in a separate PR? I think this one is separate.

Reply:
I can make the change. Which should it be?
```
TileLayer.defaultProps.fetch = window.fetch;
```
or
```
import {load} from '@loaders.gl/core';
TileLayer.defaultProps.fetch = load;
```

Reply:
I assume the latter and updated this PR.

Reply:
Check out the definition in the base layer:
https://github.com/visgl/deck.gl/blob/c8702ae134e0598b2fdca03642744f25e9de593b/modules/core/src/lib/layer.js#L66-L90

Reply:
Made the update to look more like core layer's fetch, but without the resource manager parts.